### ISSUE TITLE
Question regarding training time and resources

### ISSUE THREAD

**Issue by Meteord** [aqlaboratory/openfold#99]
Hi :),

First of all, thanks for your great work!

I'm trying to train _openfold_ (with just 10 evoformer blocks)  on _proteinnet_ with two _quadro rtx 8000_ GPUs.

My loss isn't increasing much after about 5 epochs, and the results are far away from being useful:
![loss](https://user-images.githubusercontent.com/5181036/166825403-17bf2acc-bd4f-4df5-82d7-f9648d9803d8.png)

Now I am wondering if I can expect the loss to converge at some time in the near future with more training. Or does it seem as I am doing something completely wrong? I'm asking because the original _alphafold_ training used a lot of hardware resources (several weeks with about 128 TPUv3) which I do not currently own.

Which hardware did you use for training and how long did it take until you got useful results? 

Thanks in advance!

**Comment by gahdritz** [comment]
We're using approx. 45 A100s ATM, and it takes a very long time (weeks) to get good results. Our loss curves look pretty much the same---they all have a rapid initial learning phase followed by an extremely gradual (but steady) decrease in the loss. With just 2 GPUs, it might be worth considering trying to finetune the AlphaFold weights instead of training from scratch.
